### PR TITLE
Add `multiple()` functionality to AttachAction and AssociateAction

### PR DESCRIPTION
**Pull request by W1-PopelierE**
## Description
In discussion #11894, I proposed adding ﻿->multiple() to the AttachAction. @danharrin approved, with the condition of adding it to AssociateAction as well. This enhancement enables attaching or associating using a multiple select.

- [x] Visual changes (if any) are shown using screenshots/recordings of before and after.

Not really a new style but a screenshot won't hurt.
image

## Code style

- [x] `composer cs` command has been run.

## Testing

- [x] Changes have been tested.

I was unsure where to add the tests for AttachActions and AssociateActions. However, since there were separate tests for BulkActions, I decided to make these tests separate as well.
## Documentation

- [x] Documentation is up-to-date.

### CODE REVIEW

File: packages/panels/docs/03-resources/07-relation-managers.md [hunk 1]
```diff
@@ -337,6 +337,17 @@ AttachAction::make()
     ->recordSelectSearchColumns(['title', 'description'])
 ```
 
+### Multi-select
```
Comment (danharrin) [inline packages/panels/docs/03-resources/07-relation-managers.md]:
```suggestion
### Attaching multiple records
```

File: packages/panels/docs/03-resources/07-relation-managers.md [hunk 2]
```diff
@@ -440,6 +451,17 @@ AssociateAction::make()
...
```
Comment (danharrin) [inline packages/panels/docs/03-resources/07-relation-managers.md]:
```suggestion
### Associating multiple records
```

File: packages/tables/.stubs.php
Comment (danharrin) [inline packages/tables/.stubs.php:1]: I think some of these testing helpers probably aren't required and can be replicated easily:

`->assertTableActionExists(AttachAction::class)`
`->assertTableActionDoesNotExist(AttachAction::class)`
`->assertTableActionExists(AssociateAction::class)`
`->assertTableActionDoesNotExist(AssociateAction::class)`

And I think we can alter `assertTableActionExists()` to accept a second parameter, `$checkActionUsing`, inspired by `assertTableColumnExists()`. Then you could do:

`->assertTableActionExists(AttachAction::class, fn (AttachAction $action) => $action->isMultiple())` to ensure it is multiple, which is more flexible

**Review by danharrin — CHANGES_REQUESTED** [review]
Thanks for the effort here! I have some comments about the testing helpers introduced

File: packages/tables/.stubs.php
Comment (W1-PopelierE) [reply packages/tables/.stubs.php:1]: I made some changes, not really sure if this is what you meant though...

**Comment by danharrin** [comment]
Thanks!

**Pull request merged**